src/components/yacht/YachtDetail.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import './YachtDetail.css';
+
+function YachtDetail() {
+  const yachtData = {
+    date: '2023-08-23T10:00:00Z',
+    city: 'Miami',
+    user_id: 1,
+    yacht_id: 1,
+    created_at: '2023-08-20T08:00:00Z',
+    updated_at: '2023-08-20T08:00:00Z',
+    image_path: '/path/to/image1.jpg',
+    yacht_name: 'Luxury Cruiser',
+    price: 1500,
+    duration: '4 hours',
+    yacht_model: 'Model X',
+  };
```

Comment:
Please try to resolve or make the API call (backend-repo) first and then implement in frontend app. So, that we can use redux store and actual data from API without using fake or static data.